queue/abstract.lua

```diff
@@ -668,13 +672,20 @@ local function switch_in_replicaset(replicaset_mode)
         box.space._queue_taken_2_mgr:insert(tuple)
     end
 
-    box.space._queue_taken_2:drop()
-    box.space._queue_taken_2_mgr:rename('_queue_taken_2')
+    if is_ddl_txn_supported then
```

Comment:
Please add a comment here. Because for a new developer it will look like:
```
if x then
    do some action
end
...

if not x then
    perform the same action that would be done if x were true
end
```

Reply:
Added/